src/namespaces/PlaybackNamespace.js

```diff
@@ -150,6 +151,18 @@ export default class PlaybackNamespace extends GMusicNamespace {
     document.querySelector(controlsSelectors.repeat).click();
   }
 
+  isPodcast() {
+    return document.querySelector(podcastSelectors.podcast).classList.contains('podcast');
+  }
+
+  rewindTen() {
+    document.querySelector(controlsSelectors.rewindTen).click();
```

Comment:
As I left my last comment I realized that this will throw errors on non-US GPM accounts as the button doesn't exist.  We should check that the element exists before attempting to click it in this case